docs/CONTRIBUTING-id.md

```diff
@@ -1,68 +1,67 @@
+    - *Playgrounds*: Situs web interaktif, permainan, atau perangkat lunak desktop untuk belajar pemrograman. Anda dapat menulis, mengkompilasi (atau menjalankan), dan membagikan potongan kode. Playgrounds seringkali memperbolehkan Anda untuk membuat salinan (fork) dan membebaskan Anda untuk bermain dengan kodenya.
     - *Podcast dan Screencasts*: Podcast dan Screencasts.
-    - *Kumpulan Masalah & Pemrograman Kompetitif*: Situs web atau perangkat lunak yang memberi Anda kesempatan untuk menguji keterampilan pemrograman Anda dengan memecahkan masalah sederhana atau kompleks, dengan atau tanpa tinjauan kode dan dengan atau tanpa membandingkan kinerja dengan orang lain Pengunjung situs.
+    - *Kumpulan Masalah & Pemrograman Kompetitif*: Situs web atau perangkat lunak yang memungkinkan Anda mengukur kemampuan pemrograman Anda dengan menyelesaikan masalah-masalah sederhana atau kompleks, dengan atau tanpa tinjauan kode, dengan atau tanpa membandingkan hasilnya dengan pengguna lain.
 
-4. Pastikan Anda mengikuti [Guidelines](#guidelines) dan hormati [Markdown Formatting](#formatting) dari file.
+4. Pastikan Anda mengikuti [Guidelines di bawah](#guidelines) dan menghormati [Formatting Markdown](#formatting) dari setiap file.
 
-5. GitHub Actions akan menjalankan tes untuk memastikan bahwa **daftar diurutkan berdasarkan abjad dengan benar** dan bahwa **aturan pemformatan telah diikuti**. **Pastikan** perubahan Anda lulus tes ini.
+5. GitHub Actions akan menjalankan pengujian untuk **memastikan bahwa daftar yang Anda buat diurutkan secara alfabetis** dan **mengikuti aturan format**. **Pastikan** untuk memeriksa bahwa perubahan yang Anda buat lulus uji coba tersebut.
 
 
 <!----><a id="guidelines"></a>
 ### Pedoman
 
-- Pastikan sebuah buku benar-benar gratis. Periksa kembali jika perlu. Ini membantu administrator jika Anda menjelaskan dalam PR Anda mengapa menurut Anda buku tersebut gratis.
-- Kami tidak merekam file yang ada di Google Drive, Dropbox, Mega, Scribd, Issuu atau platform unggah file lainnya sebanding.
-- Masukkan tautan dalam urutan abjad, as described [below](#alphabetical-order).
-- Selalu pilih tautan dari sumber otoritatif (yaitu, situs web penulis lebih baik daripada situs web editor, yang pada gilirannya akan lebih baik daripada situs web pihak ketiga)
-    - tidak ada platform hosting file (termasuk tautan ke Dropbox, Google Drive, dll.).
-- Tautan `https` harus selalu lebih disukai daripada tautan `http` - selama tautan tersebut mengarah ke domain dan konten yang sama.
-- Garis miring harus dihapus pada domain root: `http://example.com` alih-alih `http://example.com/`
+- Pastikan bahwa buku yang Anda tambahkan benar-benar gratis. Periksa dua kali jika perlu. Ini akan membantu para admin jika Anda memberikan komentar di PR mengenai alasan Anda menganggap buku tersebut gratis.
+- Kami tidak menerima file yang dihosting di Google Drive, Dropbox, Mega, Scribd, Issuu, dan platform unggah file serupa lainnya.
```

Comment:
I though the word 'dihosting' should be changed to 'diunggah'.

Reply:
Have you read the original [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md) docs?

It simply says `hosted`, not `uploaded` which means to `diunggah`.

So I guess there's no change needed.

Reply:
just curious, why not to be better translated? 
I think this guide for indonesian people, used the good sentences is better instead terms.

I guess it's can be translated like
- Kami tidak menerima file dari Google Drive, ...
- Kami tidak menerima file yang bersumber dari Google Drive, ...

Reply:
Hey, that's actually even nicer. Didn't think of that.
Let me change it that way.

Reply:
Thank you for your suggestion, that definitely looks better.